envs/bcftools/bcftools-1.10.2.yaml

```diff
@@ -32,5 +32,5 @@ dependencies:
   - tk=8.6.10
   - xz=5.2.5
   - zlib=1.2.11
-prefix: /home/prasathp/miniconda3/envs/test-bcftools
+ 
```

Comment:
here and in other yaml files, can you please make sure this trailing space is not included? It is added to the majority of the yaml files